Jellyfin.Server.Implementations/Devices/DeviceManager.cs

```diff
@@ -37,6 +39,24 @@ public DeviceManager(IDbContextFactory<JellyfinDbContext> dbProvider, IUserManag
         {
             _dbProvider = dbProvider;
             _userManager = userManager;
+            _devices = new ConcurrentDictionary<int, Device>();
+            _deviceOptions = new ConcurrentDictionary<string, DeviceOptions>();
```

Comment:
Would be better with a tuple i.e. `ConcurrentDictionary<int, (Device, DeviceOptions?)>()` as they are very tightly coupled

Reply:
Well now it turns out it is not, as different devices could have same DeviceId for whatever reason and store them separately is easier for such many to one relationship.